addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -175,12 +175,10 @@ const FormEditor = options.Class.extend({
         if (field.custom && !field.string) {
             params.field.string = field.name;
         }
-        if (field.type === "description") {
```

Comment:
Note: here is my hypothesis on why the author wrote this `if`:
The name of the template used to render the field is `"website.form_field_" + field.type`. As the name of the template in charge of rendering the description is `website.form_field_description`, the author probably thought that this template should only be called when `field.type === "description"`. This is not the case; the template `website.form_field_description` is always called when rendering a field (through `website.form_field`) but is only relevant if `default_description` is set.